recipe/build.sh

```diff
@@ -3,6 +3,10 @@
 export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
 export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
 
+if [ -n "$OSX_ARCH" ] ; then
```

Comment:
I certainly don't have any strong opinions about the best way to do this. I guess the `$OSX_ARCH` test doesn't spawn a subprocess? That's just about the weakest possible reason in the world to prefer it, though.